code/modules/power/cell.dm

```diff
@@ -10,6 +10,8 @@
 	throw_speed = 2
 	throw_range = 5
 	w_class = 2
+	var/part_type = "cell" //I blame vehicles
```

Comment:
It was an attempt to stick all the parts into a single istype chain, but if you insist on me seperating and removing the variables, i'll go ahead and do so.